<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kitton Informática - Configuração</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Imagens/Logomarca Ícone.ico</normaloff>Imagens/Logomarca Ícone.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(205,240,255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 170, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="botaoSelecionar">
       <property name="geometry">
        <rect>
         <x>1500</x>
         <y>590</y>
         <width>170</width>
         <height>60</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(0, 0, 127);
	color: rgb(255, 255, 255);
border-radius: 20px;
}
QPushButton:hover
{
   background-color:white;
	color: rgb(0, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>Selecionar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="botaoVoltar">
       <property name="geometry">
        <rect>
         <x>1500</x>
         <y>830</y>
         <width>170</width>
         <height>60</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(0, 0, 127);
	color: rgb(255, 255, 255);
border-radius: 20px;
}
QPushButton:hover
{
   background-color:white;
	color: rgb(0, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>Voltar</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>230</y>
         <width>831</width>
         <height>491</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>codigo</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>nome</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>categoria</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>fornecedor</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>custo_fabrica</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>funcionario</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>preco</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="botaoAlterar">
       <property name="geometry">
        <rect>
         <x>1300</x>
         <y>660</y>
         <width>170</width>
         <height>60</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(0, 0, 127);
	color: rgb(255, 255, 255);
border-radius: 20px;
}
QPushButton:hover
{
   background-color:white;
	color: rgb(0, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>Alterar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="botaoPesquisa">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>830</y>
         <width>211</width>
         <height>60</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(0, 0, 127);
	color: rgb(255, 255, 255);
border-radius: 20px;
}
QPushButton:hover
{
   background-color:white;
	color: rgb(0, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>Pesquisa</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>301</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
       </property>
       <property name="text">
        <string>Campo de Pesquisa</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>220</y>
         <width>20</width>
         <height>701</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>770</y>
         <width>321</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_3">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>770</y>
         <width>841</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="botaoDeletar">
       <property name="geometry">
        <rect>
         <x>1700</x>
         <y>660</y>
         <width>170</width>
         <height>60</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(0, 0, 127);
	color: rgb(255, 255, 255);
border-radius: 20px;
}
QPushButton:hover
{
   background-color:white;
	color: rgb(0, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>Deletar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>1380</x>
         <y>190</y>
         <width>401</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
       </property>
       <property name="text">
        <string>Campo de Administração</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>1390</x>
         <y>280</y>
         <width>392</width>
         <height>274</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Nome</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="nomeAlterar">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="codigoPesquisa">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="fornecedorAlterar">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Fornecedor</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="custo_fabAlterar">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Funcionario</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Custo Fabrica</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="precoAlterar">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Categoria</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="preco">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Preco</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="categoriaAlterar">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
          </property>
          <item>
           <property name="text">
            <string>Selecione</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HD</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SSD</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Memória RAM</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Placa-mãe</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mouse</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Placa de Rede</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cabo de rede</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Teclado</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Monitor</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gabinete</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pendrive</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fonte de Alimentação</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cabo VGA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cabo HDMI</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="funcionarioAlterar">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
          </property>
          <item>
           <property name="text">
            <string>Selecione</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Kamily Kathleen</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bruno Rangel</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ana Caroline</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fernanda Tayná</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Matheus Souza</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Codigo Pesquisa</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>15</y>
         <width>1421</width>
         <height>111</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>50</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius:40px;</string>
       </property>
       <property name="text">
        <string>Gerenciamento de Produtos</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="img3">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>20</y>
         <width>101</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius:40px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>Imagens/Logomarca.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="img3_2">
       <property name="geometry">
        <rect>
         <x>1540</x>
         <y>20</y>
         <width>101</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius:40px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>Imagens/Logomarca.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="categoria">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>460</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
       <item>
        <property name="text">
         <string>Selecione</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SSD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Memória RAM</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Placa-mãe</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mouse</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Placa de Rede</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cabo de rede</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Teclado</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Monitor</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gabinete</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pendrive</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fonte de Alimentação</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cabo VGA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cabo HDMI</string>
        </property>
       </item>
      </widget>
      <widget class="Line" name="line_4">
       <property name="geometry">
        <rect>
         <x>1240</x>
         <y>220</y>
         <width>20</width>
         <height>701</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="Line" name="line_5">
       <property name="geometry">
        <rect>
         <x>1280</x>
         <y>770</y>
         <width>591</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="botaoAtualizar">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>830</y>
         <width>100</width>
         <height>60</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(0, 0, 127);
	color: rgb(255, 255, 255);
border-radius: 20px;
}
QPushButton:hover
{
   background-color:white;
	color: rgb(0, 0, 0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Imagens/atualizar.png</normaloff>Imagens/atualizar.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
      <zorder>layoutWidget</zorder>
      <zorder>botaoSelecionar</zorder>
      <zorder>botaoVoltar</zorder>
      <zorder>tableWidget</zorder>
      <zorder>botaoAlterar</zorder>
      <zorder>botaoPesquisa</zorder>
      <zorder>line</zorder>
      <zorder>line_2</zorder>
      <zorder>line_3</zorder>
      <zorder>botaoDeletar</zorder>
      <zorder>label_3</zorder>
      <zorder>label_8</zorder>
      <zorder>label_11</zorder>
      <zorder>img3</zorder>
      <zorder>img3_2</zorder>
      <zorder>categoria</zorder>
      <zorder>line_4</zorder>
      <zorder>line_5</zorder>
      <zorder>botaoAtualizar</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
